tracee/printer.go

```diff
@@ -63,10 +63,11 @@ type Event struct {
 	EventName       string        `json:"eventName"`
 	ArgsNum         int           `json:"argsNum"`
 	ReturnValue     int           `json:"returnValue"`
+	ArgsNames       []string      `json:"argsNames"`
```

Comment:
I think we should not just add the argnames array to the Event type here, but instead combine it with the args array, probably as a `map[string]interface{}`. The reason we created this Event type is that we could decouple the internal data structures (context, []args) that are required to process events from the external data type (Event) that is used to communicate events externally (gob) and for printing (printer). This is a good opportunity to take use the Event struct to adapt the internal representation (2 arrays) which were convenient for processing but not for consuming, and adjust it to be convenient for consuming (one map).

Reply:
I'm handling this in some different way in my next PR which is based on this one. We can change this internal representations after these will be merged